settings/contentstructuremenu.ini

```diff
@@ -36,9 +36,8 @@ MaxNodes=150
 #    SortBy[]
 #    SortBy[]=name/ascending
 #    SortBy[]=published/descending
-# if set to "false" - default sorting will be implemented
-# (which is defined for each node)
-SortBy=false
+# if not set, it uses the sorting defined for each node (from the Ordering tab)
+SortBy=[]
```

Comment:
That is not a INI array value.
Emptying and setting the value as array is as documented `SortBy[]`.
Appending is done by means of `SortBy[]=name/ascending`

Reply:
Thank you @andrerom!